src/lib/core/fiber.h

```diff
@@ -340,6 +340,7 @@ fiber_set_cancellable(bool yesno);
 
 /**
  * Set fiber to be joinable (false by default).
+ * \param fiber to be joinable
```

Comment:
'fiber to (un)set the joinable property' I would say. Because the second parameter can be `false`.

Reply:
Updated, thanks!